Makefile.am

```diff
@@ -1,7 +1,8 @@
 SUBDIRS = server client data data/init.d data/systemd
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
-EXTRA_DIST = autogen.sh README.md hatohol.spec
+EXTRA_DIST = autogen.sh README.md hatohol.spec \
+			 COPYING.GPLv3 COPYING.LGPLv3
```

Comment:
Why is COPYING.GPLv3 necessary ?
All source files are distributed under LGPLv3, aren't they ?